src/parse.ros2.test.ts

```diff
@@ -268,12 +274,14 @@ describe("parseMessageDefinition", () => {
             type: "bool",
             isConstant: true,
             value: true,
+            valueText: "1", // This should be "True" (https://github.com/foxglove/rosmsg/issues/13)
```

Comment:
I don't understand this comment.

Reply:
valueText is "1", but that's not correct. valueText should be "True" as explained in the linked issue.